diff --git a/os-probes/mounted/common/90linux-distro b/os-probes/mounted/common/90linux-distro
index 9544531..7476b5f 100755
--- a/os-probes/mounted/common/90linux-distro
+++ b/os-probes/mounted/common/90linux-distro
@@ -137,6 +137,9 @@
 	elif [ -e "$dir/etc/devuan_version" ]; then
 		short="Devuan"
 		long="$(printf "Devuan GNU/Linux (%s)\n" "$(cat "$dir/etc/devuan_version")")"
+       elif [ -e "$dir/etc/entropy-release" ];then
+		short="Entropy"
+		long="$(printf "Entropy GNU/Linux (%s)\n" "$(cat "$dir/etc/entropy-release")")"
 	else
 		short="Linux"
 		long="unknown Linux distribution"
